Bio/PDB/PDBList.py

```diff
@@ -320,42 +420,61 @@ def get_seqres_file(self, savefile='pdb_seqres.txt'):
     PDBList.py obsol  <pdb_path> [options]   - write all obsolete PDB
                                                entries to local pdb tree.
     PDBList.py <PDB-ID> <pdb_path> [options] - retrieve single structure
+    PDBList.py (<PDB-ID1>,<PDB-ID2>,...) <pdb_path> [options] - retrieve a set
+                                               of structures
 
     Options:
-       -d   A single directory will be used as <pdb_path>, not a tree.
-       -o   Overwrite existing structure files.
+       -d       A single directory will be used as <pdb_path>, not a tree.
+       -o       Overwrite existing structure files.
+       -pdb     Downloads structures in PDB format
+       -xml     Downloads structures in PDBML (XML) format
+       -mmtf    Downloads structures in mmtf format
+
+    Maximum one format can be specified simultaneously (if more selected, only
+    the last will be considered). By default (no format specified) structures are
+    downloaded as PDBx/mmCif files.
     """
     print(doc)
 
+    file_format = "mmCif"
+    overwrite = False
+
     if len(sys.argv) > 2:
```

Comment:
Reworking this module could be a good time to switch to argparse; this would reduce the complexity of the control flow.